.github/workflows/hardware-smoke.yml

```diff
@@ -267,4 +263,4 @@ jobs:
           echo "DMC RTT logs:"
           python3 ./scripts/dmc_rtt.py -n --openocd /opt/tenstorrent/bin/openocd-rtt
           echo "SMC RTT logs:"
-          python3 ./scripts/smc_rtt.py -n --openocd /opt/tenstorrent/bin/openocd-rtt
+          python3 ./scripts/smc_rtt.py -n
```

Comment:
SMC needs to use openocd RTT, only BMC will be safe with the default executable

Reply:
Oh sorry - I had this backwards. Good catch

Reply:
Fixed!